emulators/mame/Portfile

```diff
@@ -125,9 +159,15 @@ depends_run-append  \
 # Build Options
 #------------------------------------------------------------------------------
 
-use_parallel_build  yes
+# Note: ARM64 support untested
+#supported_archs     ppc ppc64 i386 x86_64 arm64
+supported_archs     ppc ppc64 i386 x86_64
 
+use_parallel_build  yes
 use_configure       no
+configure.ccache    yes
+
+build.cmd           ${prefix}/bin/gmake
```

Comment:
you almost never need this, unless you're building on Tiger, which has a really old make program. If you really do need this, don't forget to have a build dep on port:gmake somewhere so it actually exists.

Reply:
This port utilizes newer gmake option `--trace`, not available in Apple's version.

I'll add a comment to clarify that.